### ISSUE TITLE
Add Image to the devices

### ISSUE THREAD

**Issue by jheyjheyjhey** [shakee93/fonoapi#39]
Hi, currently using your API on my little project. Can we have image for the device? Let me know help is necessary, I can volunteer.

Cheers!

**Comment by alexandruspataru** [comment]
@jheyjheyjhey I already have 3-4000+ images, all of them square and 500x500px size, however, think that this cannot be done by the API provider as it comes with a lot of cloud space + bandwidth.

**Comment by isaumya** [comment]
Yes I'm also looking for this feature, also as this is an open source project, I'm sure that many company provide free hosting for open source projects.

**Comment by xitude** [comment]
> @jheyjheyjhey I already have 3-4000+ images, all of them square and 500x500px size, however, think that this cannot be done by the API provider as it comes with a lot of cloud space + bandwidth.

Do you still have this? It would be useful for my dataset
